tests/test_hear_server_activate.py

```
@@ -120,5 +122,72 @@ def run_server() -> None:
+
+	def test_initialization_with_valid_address(self):
+		"""
+		Test multicast server initialization with a valid address.
+
+		Verifies that:
+			1. The server instance is of the correct type (McastServer).
+			2. The server instance is also recognized as a UDPServer.
+			3. Cleanup is performed correctly after initialization.
+		"""
+		server = multicast.hear.McastServer(('224.0.0.1', 12345), None)
+		self.assertIsInstance(server, multicast.hear.McastServer)
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertIsInstance`

Replace `assertIsInstance(...)` with `assert ...`

(PT009)